tiabobia-honolulu.md

```diff
@@ -0,0 +1,16 @@
+# Honolulu, Hawaii
```

Comment:
@tiabobia  - How do you like living in Hawaii?  Which restaurant would you pick most highly?